shells/android/java/arcs/storage/service/IResultCallback.aidl

```diff
@@ -16,9 +16,9 @@ import arcs.crdt.parcelables.ParcelableCrdtException;
 /** Mechanism allowing an asynchronous response from the StorageService. */
 interface IResultCallback {
     /**
-     * Handles success/failure of an async op.<br><br>
+     * Called to signal a successful/erroneous result from the StorageService or ServiceStore.
      *
-     * {@param exception} is nullable.
+     * <p>{@code exception} will be {@code null} when the result indicates success.
```

Comment:
I know you're planning to come back to fix these up properly later, but it's still bothering me.

```
@param exception will be {@code null} when the ...
```

Reply:
Ah, ok.. I'll use this approach.